lab_5_scrapper/scrapper_config.json

```diff
@@ -1,9 +1,13 @@
 {
-    "seed_urls": [],
-    "headers": {},
-    "total_articles_to_find_and_parse": 0,
-    "encoding": "",
-    "timeout": 0,
+    "seed_urls": ["https://www.gazetavechorka.ru/news/proisshestviya"],
```

Comment:
will it be enough to collect 100 articles?

Reply:
fixed